testing/bsg_dmc/tb_top/bsg_dmc_tb_top_signals.vh

```diff
@@ -0,0 +1,84 @@
+  wire bsg_tag_s        dmc_reset_tag_lines_lo;
```

Comment:
Okay... my rationale behind putting it in a separate file was to de-clutter the testbench top file which would already have a lot of content... Since these wires and regs would already be used during module instanciations, thought it better to keep it out of the TB top file.
Let me know if you still think it better to put these in the TB top.

Reply:
removed separate file and added everything to top TB file